config/crd/bases/_.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
I have tried to debug this issue of _.yaml being generated but don't have a clue why this happens. But please remove this from the PR.

Reply:
Yeah, that slipped in.